website/src/css/customTheme.scss

```
@@ -182,6 +182,18 @@ hr {
       top: -1px;
       position: relative;
     }
+
+    .hermes-logo {
+      float: right;
+      margin: -30px 4px 0;
+    }
+
+    @media screen and (max-width: 960px) {
+      .hermes-logo {
+        margin: -20px 4px 0;
```

```suggestion
        margin: 0 4px 30px;
```
Spacing fix for the mobile.